### PR TITLE
Updated ProfileMeasurementValue types

### PR DESCRIPTION
**Pull request by stefanosiano**
## :scroll: Description
Changed ProfileMeasurementValue types: timestamp is now a string and value is a double
Added a check against negative timestamps

## :bulb: Motivation and Context
Relay added a stronger type check on the profile measurements. This caused profiles to be rejected. While the issue has already been addressed in relay, it's better to update the schema to what relay expects.

## :green_heart: How did you test it?
Updated unit test

## :pencil: Checklist

- [x] I reviewed the submitted code
- [x] I added tests to verify the changes
- [x] I updated the docs if needed
- [x] No breaking changes

### CODE REVIEW

**Review by markushi — APPROVED** [review]
Looks good to me! Do we need to migrate other components (e.g. hybrid SDKs) or wait for a backend release (e.g. on relay) before this can be merged?

**Comment by stefanosiano** [comment]
> Looks good to me! Do we need to migrate other components (e.g. hybrid SDKs) or wait for a backend release (e.g. on relay) before this can be merged?

nope: this was the expected format from the beginning, but relay was previously more "permissive" and profiles were ingested.
This change is because relay was updated to check strictly the types, but that check was temporary reverted to allow the sdk to update the code

**Pull request merged**
